PactNet/Mocks/MockHttpService/MockProviderService.cs

```diff
@@ -27,28 +27,30 @@ public class MockProviderService : IMockProviderService
             Func<Uri, IHttpHost> hostFactory,
             int port,
             bool enableSsl,
+            bool enableIpv6,
             Func<Uri, AdminHttpClient> adminHttpClientFactory)
         {
             _hostFactory = hostFactory;
-            BaseUri = new Uri($"{(enableSsl ? "https" : "http")}://localhost:{port}");
+            BaseUri = new Uri($"{(enableSsl ? "https" : "http")}://{(enableIpv6 ? "localhost" : "127.0.0.1")}:{port}");

```

Comment:
This is also a breaking change - I'm not sure why ipv6 should always host using the hostname but ipv4 would use the loopback IP? Wouldn't it be more consistent to use `::1` for ipv6 if the user wants to use the IP instead of the hostname alias?

Either way, I think that would be a choice that should be left up to the user. At the moment this is a potentially breaking change for them.